drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -151,7 +151,13 @@ def self.options_for_type(pick_list_type, user:, relation_id: nil)
       end
     end
 
+    def self.all_unit_types
+      Hmis::UnitType.order(:description, :id).map(&:to_pick_list_option)
```

Comment:
Is there, or should there be an "active" or "inactive" flag/scope on UnitType?

Reply:
There is not. The Hmis::ProjectUnitTypeMapping maps Projects=>Allowed Unit Types and that table does have a active/inactive flag, but the Unit Types themselves don't. I think it's ok for now we do not have a use case for it.